app/src/main/java/com/bogdan/codeforceswatcher/features/news/shared/PostContentView.kt

```diff
@@ -0,0 +1,47 @@
+package com.bogdan.codeforceswatcher.features.news.shared
+
+import androidx.compose.foundation.background
+import androidx.compose.foundation.layout.*
+import androidx.compose.material.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.draw.clip
+import androidx.compose.ui.text.style.TextOverflow
+import androidx.compose.ui.unit.dp
+import com.bogdan.codeforceswatcher.components.compose.theme.AlgoismeTheme
+
+@Composable
+fun PostContentView(
+    title: String,
+    content: String,
+    handle: String,
+    avatar: String,
+    rank: String?,
+    modifiedAt: Long,
```

Comment:
Not sure, but maybe `modifiedAt` can be made `optional`, so you can pass `null` when `isModified = false`.

Reply:
`isModified` is passed to correctly display the "created at"/"modified at" text for the last modified time, so `modifiedAt` is always passed.

Reply:
OK.